docs/reference/glossary.asciidoc

```diff
@@ -450,6 +456,12 @@ in the <<glossary-mapping,mapping>>.
 // end::routing-def[]
 --
 
+[[glossary-searchable-snapshot]] searchable snapshot ::
+// tag::searchable-snapshot-def[]
+A <<glossary-snapshot, snapshot>> of an index or data stream that resides in a remote data store 
```

Comment:
I wonder if the "searchable snapshot" object is the index or the snapshot? We talked about "snapshot-backed index", which is clearly the index. I think of it as the index that makes the snapshot searchable.

We could also define "searchable snapshot index/indices" here instead.

I find the current definition here slightly confusing in that any snapshot can be made searchable.